phpunit/class-gutenberg-rest-api-test.php

```diff
@@ -119,4 +119,16 @@ function test_viewable_field_without_context() {
 		$result   = $response->get_data();
 		$this->assertFalse( isset( $result['viewable'] ) );
 	}
+
+	/**
+	 * Should include relevant data in the 'theme_supports' key of index.
+	 */
+	function test_theme_supports_index() {
+		$request  = new WP_REST_Request( 'GET', '/' );
+		$response = rest_do_request( $request );
+		$result   = $response->get_data();
+		$this->assertTrue( isset( $result['theme_supports'] ) );
+		$this->assertTrue( isset( $result['theme_supports']['formats'] ) );
+		$this->assertTrue( in_array( 'standard', $result['theme_supports']['formats'] ) );
```

Comment:
It'd be good to test that post formats that aren't specified by the theme are not in the results, too.

Reply:
Given post formats can vary from theme to theme, and the theme active in the test suite is uncertain, can you suggest what you'd like me to test?

Reply:
On second thoughts, I don't know that it needs testing. I'd been looking at WPCOM's API, which tries to avoid loading the theme code if it can, so was considering how to test for that. But, that's not really an issue for Core.